Vagrantfile

```diff
@@ -333,32 +335,32 @@ Vagrant.configure('2') do |config|
             else
                 node.vm.box = box[:name]
 
-                if box.key?('version')
+                if box.key?(:version)
```

Comment:
Thanks for fixing all these. Apparently my regex wasn't as good as I thought ;o)

Reply:
I found some more issues